app/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import { render } from 'react-dom';
-import { AppContainer } from 'react-hot-loader';
+import { AppContainer as ReactHotAppContainer } from 'react-hot-loader';
 import Root from './containers/Root';
 import { configureStore, history } from './store/configureStore';
 import './app.global.css';
 
 const store = configureStore();
 
+const AppContainer = process.env.PLAIN_HMR
+  ? React.Fragment
```

Comment:
Would prefer importing `Fragment` explicitly.
```suggestion
  ? Fragment
```